### ISSUE TITLE
Maps not Orion v1.0 compartible

### ISSUE THREAD

**Issue by timfam** [rwatts3/orionjs-contrib#5]
Will this be ported over to Orion V1.0?

**Comment by rwatts3** [comment]
Yes it will, Was actually in the middle of working on it, want to ensure 1.1 doesn't introduce any breaking changes. I will be updating within the next couple weeks. Possibly sooner.

**Comment by timfam** [comment]
That will be nice

**Comment by ignacy130** [comment]
Hi, I'm also really interested in maps! ;) Any way to help you?

**Comment by rwatts3** [comment]
Thank you.  I was supposed to have fm this done a while back and got side
tracked.  I'll work on it this weekend.

On Friday, August 28, 2015, Ignacy <notifications@github.com> wrote:

> Hi, I'm also really interested in maps! ;) Any way to help you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rwatts3/orionjs-contrib/issues/5#issuecomment-135726164>
> .
>


-- 
Ryan Watts
E.ryandwatts@gmail.com
C.520-442-8962
F.520-437-0914
IN. https://www.linkedin.com/in/ryandwatts

**Comment by fr0zen** [comment]
Hey. @rwatts3, what happened with the project?
